plugins/python-build/bin/python-build

```diff
@@ -1492,15 +1492,20 @@ has_broken_mac_openssl() {
 }
 
 use_homebrew_openssl() {
-  local ssldir="$(brew --prefix openssl 2>/dev/null || true)"
-  if [ -d "$ssldir" ]; then
-    echo "python-build: use openssl from homebrew"
-    export PKG_CONFIG_PATH="$ssldir/lib/pkgconfig/:${PKG_CONFIG_PATH}"
-    export CPPFLAGS="-I$ssldir/include ${CPPFLAGS}"
-    export LDFLAGS="-L$ssldir/lib ${LDFLAGS}"
+  local ssl11dir="$(brew --prefix openssl@1.1 2>/dev/null || true)"
```

Comment:
This lookup behaviour might be conflict with `openssl` lookup in `compute_sha2()`. But it would work sufficiently as long as there is some `openssl` available in `PATH` 🤔 

https://github.com/pyenv/pyenv/blob/v1.2.9/plugins/python-build/bin/python-build#L253

Reply:
I love you. ❤️

Reply:
I just noticed the compute_sha2 works a bit differently than the compute_md5 right below it (and, assuming that it works, the latter does not need to use brew to locate openssl).

Reply:
I also wondered why both compute_sha2 and compute_md5 try sha256sum and md5sum commands last (after trying openssl).

Reply:
@yyuu ^^^